configure.ac

```diff
@@ -2892,6 +2892,9 @@ if test "$with_uhdr" != 'no'; then
     failed=0
     passed=0
 
+    PKG_CHECK_MODULES([UHDR],[libuhdr >= 1.0],[have_uhdr=yes],[have_uhdr=no])
```

Comment:
See https://github.com/google/libultrahdr/issues/20#issuecomment-2036781883

Reply:
Looks like both the library name and pkgconf file settled on "libuhdr" upstream, so this is ok.